NeoML/include/NeoML/NeoMLCommon.h

```diff
@@ -47,12 +49,14 @@ if( !( expr ) ) { \
 
 #else // Release
 
-#define NeoAssert( expr ) \
+#define NeoAssertMsg( expr, text ) \
```

Comment:
The same expectation. Unrelated changes has not to be included in Pull request

Reply:
Without this update we cannot make a readable warning for user, if it uses new feature wrong
```
NeoAssertMsg(referenceDnnRegister.referenceCounter == 0,
		"learning restricted if reference dnn exist");
```

Reply:
Let's leave it as is here.
Next time we make separate PR for NeoAssert with message, and after it PR with feature,  that requires new NeoAssert.